src/main/java/com/ltbc/swarmrestapi/service/beesinfo/Food.java

```diff
@@ -0,0 +1,7 @@
+package com.ltbc.swarmrestapi.service.beesinfo;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record Food(boolean isFoodPresent,  String desc) {
+}
```

Comment:
For bee food data, maybe we can also track blooming patterns for flowers in the area 🤔 using a different API

Reply:
We could but we can put a ticket in the backlog to enhance these APIs later. I think for the first example / first API this one should be fine and from here we can expand and create more APIs

Reply:
Created #18  from your comment, we can refine it later.